client/src/App.css

```diff
@@ -244,7 +214,6 @@ td {
   cursor: pointer;
   padding: 5px;
 }
-
 .centered {
   justify-items: center;
   text-align: center;
```

Comment:
I think in css one line space after every class is helps to read the code 
I don't know why you delete the line spaces here ?

Reply:
cause youasked todelete some spaces in css;)nthats the only reason

Reply:
good job Constantin, no worry, if we have a line space after a specific piece of code it helps to be more readable